api/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+install(
+  DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/iceberg"
```

Comment:
This is the wisdom from Apache Arrow, Parquet and ORC libraries where their headers should be included via `#include "arrow/xxx.h"`, `#include "parquet/xxx.h"` and `#include "orc/xxx.h"`. Otherwise, it is unclear which library it comes from when `#include "api/xxx.h"` or `#include "xxx.h"` is used.